.github/workflows/check-submodule.yml

```diff
@@ -0,0 +1,44 @@
+name: check-submodule
+
+on: [pull_request]
+
+jobs:
+  check-submodule:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout this repo
+        uses: actions/checkout@v3
+      # We need to explicitly check out the Semgrep interfaces submodule since
```

Comment:
I think you can just use with: submodules: true
so you don't need to do a separate checkout for semgrep-interfaces, and you can remove this comment about
ssh I don't really understand.

Reply:
Thanks for catching it. semgrep-pro is a private repo, but semgrep-interfaces is public, so a separate checkout should not be necessary.